includes/class-wc-cache-helper.php

```diff
@@ -107,7 +107,7 @@ public static function get_cache_prefix( $group ) {
 	 * @param string $group Group of cache to clear.
 	 */
 	public static function incr_cache_prefix( $group ) {
-		wp_cache_incr( 'wc_' . $group . '_cache_prefix', 1, $group );
+		wp_cache_set( 'wc_' . $group . '_cache_prefix', microtime(), $group );
```

Comment:
You will need to pass in true otherwise it will return a string which contains spaces which in turn will break the key